_extras/sequence_alignment.md

```diff
@@ -0,0 +1,516 @@
+Note that the lengthy tag `v6.6.0dfsg-7b1-deb_cv1` is necessary since it is not the `latest` tag.
+
+~~~
+$ docker image pull biocontainers/emboss:v6.6.0dfsg-7b1-deb_cv1
+~~~
+{: .language-bash}
+~~~
+v6.6.0dfsg-7b1-deb_cv1: Pulling from biocontainers/emboss
+478cd0aa93c0: Pull complete 
+[...]
+$ 
+~~~
+{: .output}
+
+You can verify that the image has been copied onto your computer with a listing command:
+
+~~~
+$ docker image ls biocontainers/emboss
+~~~
+{: .language-bash}
+~~~
+REPOSITORY             TAG                      IMAGE ID        CREATED       SIZE
+biocontainers/emboss   v6.6.0dfsg-7b1-deb_cv1   bc147a9dd825    2 years ago   638MB
+~~~
+{: .output}
+
+> ## Challenge: make your own updated EMBOSS image.
+>
+> During the early part of the lesson we explored creating our own Docker images thanks to a list of instruction within a `Dockerfile` document.
+> The file for a different but similar EMBOSS Docker image is available: [Dockerfile](https://hub.docker.com/r/pegi3s/emboss/dockerfile) (also shown within "Solution" below.)  
+> This information would allow you to create your own image from a newer version of Ubuntu.   
+> How would you use this information to make your own image?   
+> Could you use another Linux distributions?  
+> Find help on an earlier section of the workshop if you need, or skip this exercise for now.   
+> 
+> > ## Solution
+> > The [Dockerfile](https://hub.docker.com/r/pegi3s/emboss/dockerfile) from a similar image is shown below:
+> > ~~~
+> > FROM ubuntu:18.04
+> > LABEL emboss.version="6.6.0" \
+> >    emboss.web="http://emboss.sourceforge.net"
+> > RUN apt-get -qq update && apt-get -y upgrade && \
+>	> apt-get install -y emboss=6.6.0+dfsg-6build1
+> > ~~~
+> > {: .output}
+> > 
+> > Here are some steps that would accomplish this task:
+> > * create a copy of the Dockerfile document to be modified (for example use `nano` as a text editor.)
+> > * Check the latest version available for ubuntu, using the [official image](https://hub.docker.com/_/ubuntu) hub page
+> > * replace this new version name and tag on the Dockerfile
+> > * Run the `docker build -t ...` command
+> > * Test your image with a `docker run ...` command
+> > * Push the image to your own hub account for future retrieval
+> > 
+> > This solution uses Ubuntu, but other Linux distributions could be used. However, this would also require changing the `apt-get` command to that specific to the chosen distribution such as `yum` for CentOS.
+> {: .solution}
+{: .challenge}
+
+> ## Shared directory
+> We'll use the EMBOSS software on files shared from our local computer.
+{: .callout}
+
+Download the [`docker-intro.zip`]({{ page.root }}/files/docker-intro.zip) file. _This file can alternatively be downloaded from the `files` directory in the [docker-introduction GitHub repository][docker-introduction repository]_. Move the downloaded file to your Desktop and unzip it. It should unzip to a folder called `docker-intro`. 
```

Comment:
I would not mind that. However, there are other files there and I am not sure if I can combine it all myself. The "basic" and "sum" directories already existed so I followed that pattern. If it can be done by someone else I could prefer as I don't want to mess things up!